### ISSUE TITLE
No progress after connecting to Mysql.

### ISSUE THREAD

**Issue by SerdarMustafa1** [dannysofftie/mysql-mongo-migrate#3]

**Comment by SerdarMustafa1** [comment]
After the connection is established to mysql server and asked to press 'tab' nothing happens. 

The terminal remains in suspended animation. 

```
 System:
    OS: macOS 10.15.5
    CPU: (8) x64 Intel(R) Core(TM) i7-6920HQ CPU @ 2.90GHz
    Memory: 1.21 GB / 16.00 GB
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.14.0 - ~/.nvm/versions/node/v12.14.0/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.5 - ~/.nvm/versions/node/v12.14.0/bin/npm
  Managers:
    CocoaPods: 1.9.2 - /Users/serdarmustafa/.rvm/gems/ruby-2.6.3/bin/pod
    Gradle: 6.4.1 - /usr/local/bin/gradle
    Homebrew: 2.4.0 - /usr/local/bin/brew
    Maven: 3.6.3 - /usr/local/bin/mvn
    pip3: 20.0.2 - /usr/local/opt/python@3.8/bin/pip3
    RubyGems: 3.0.4 - /Users/serdarmustafa/.rvm/rubies/ruby-2.6.3/bin/gem
  Utilities:
    Make: 3.81 - /usr/bin/make
    GCC: 4.2.1 - /usr/bin/gcc
    Git: 2.24.3 - /usr/bin/git
    Clang: 1103.0.32.62 - /usr/bin/clang
    Mercurial: 5.4 - /usr/local/bin/hg
    Subversion: 1.10.4 - /usr/bin/svn
  Servers:
    Apache: 2.4.41 - /usr/sbin/apachectl
  Virtualization:
    Docker: 19.03.8 - /usr/local/bin/docker
  SDKs:
    iOS SDK:
      Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    Android SDK:
      API Levels: 28, 29
      Build Tools: 28.0.3, 29.0.3
      System Images: android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6241897
    Nano: 2.0.6 - /usr/bin/nano
    Sublime Text: Build 3211
    VSCode: 1.46.0 - /usr/local/bin/code
    Vim: 8.1 - /usr/bin/vim
    Xcode: 11.5/11E608c - /usr/bin/xcodebuild
  Languages:
    Bash: 3.2.57 - /bin/bash
    Java: 1.8.0_152 - /usr/bin/javac
    Perl: 5.18.4 - /usr/bin/perl
    PHP: 7.3.11 - /usr/bin/php
    Python: 2.7.16 - /usr/bin/python
    Python3: 3.8.3 - /usr/local/opt/python@3.8/bin/python3
    Ruby: 2.6.3 - /Users/serdarmustafa/.rvm/rubies/ruby-2.6.3/bin/ruby
  Databases:
    MySQL: 5.7.29 - /usr/local/opt/mysql@5.7/bin/mysql
    PostgreSQL: 12.3 - /usr/local/bin/postgres
    SQLite: 3.32.1 - /usr/local/opt/sqlite/bin/sqlite3
  Browsers:
    Chrome: 83.0.4103.106
    Firefox: 68.0
    Firefox Developer Edition: 77.0
    Safari: 13.1.1
  Monorepos:
    Lerna: 3.22.1
```
![Screenshot 2020-06-22 at 14 24 16](https://user-images.githubusercontent.com/45297391/85284837-86557e80-b498-11ea-82c5-9112a639a012.png)

**Comment by dan-kimani** [comment]
Try to press Enter key after that step. It's a reported issue that I will definitely look into

**Comment by SerdarMustafa1** [comment]
Sadly, that doesn't work for me. Looking forward to a fix.

**Comment by dan-kimani** [comment]
Rolling out fixes soon

**Comment by fufales** [comment]
Yes i have the same issue, even for ubuntu on Windows, and Ubuntu on Mac. both...

**Comment by dan-kimani** [comment]
This project has been abandoned for a while. Will be rolling out fixes as soon as I can

**Comment by zackrspv** [comment]
> After the connection is established to mysql server and asked to press 'tab' nothing happens.
> 
> The terminal remains in suspended animation.
> 
> ```
>  System:
>     OS: macOS 10.15.5
>     CPU: (8) x64 Intel(R) Core(TM) i7-6920HQ CPU @ 2.90GHz
>     Memory: 1.21 GB / 16.00 GB
>     Shell: 5.7.1 - /bin/zsh
>   Binaries:
>     Node: 12.14.0 - ~/.nvm/versions/node/v12.14.0/bin/node
>     Yarn: 1.22.4 - /usr/local/bin/yarn
>     npm: 6.14.5 - ~/.nvm/versions/node/v12.14.0/bin/npm
>   Managers:
>     CocoaPods: 1.9.2 - /Users/serdarmustafa/.rvm/gems/ruby-2.6.3/bin/pod
>     Gradle: 6.4.1 - /usr/local/bin/gradle
>     Homebrew: 2.4.0 - /usr/local/bin/brew
>     Maven: 3.6.3 - /usr/local/bin/mvn
>     pip3: 20.0.2 - /usr/local/opt/python@3.8/bin/pip3
>     RubyGems: 3.0.4 - /Users/serdarmustafa/.rvm/rubies/ruby-2.6.3/bin/gem
>   Utilities:
>     Make: 3.81 - /usr/bin/make
>     GCC: 4.2.1 - /usr/bin/gcc
>     Git: 2.24.3 - /usr/bin/git
>     Clang: 1103.0.32.62 - /usr/bin/clang
>     Mercurial: 5.4 - /usr/local/bin/hg
>     Subversion: 1.10.4 - /usr/bin/svn
>   Servers:
>     Apache: 2.4.41 - /usr/sbin/apachectl
>   Virtualization:
>     Docker: 19.03.8 - /usr/local/bin/docker
>   SDKs:
>     iOS SDK:
>       Platforms: iOS 13.5, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
>     Android SDK:
>       API Levels: 28, 29
>       Build Tools: 28.0.3, 29.0.3
>       System Images: android-28 | Google Play Intel x86 Atom
>   IDEs:
>     Android Studio: 3.6 AI-192.7142.36.36.6241897
>     Nano: 2.0.6 - /usr/bin/nano
>     Sublime Text: Build 3211
>     VSCode: 1.46.0 - /usr/local/bin/code
>     Vim: 8.1 - /usr/bin/vim
>     Xcode: 11.5/11E608c - /usr/bin/xcodebuild
>   Languages:
>     Bash: 3.2.57 - /bin/bash
>     Java: 1.8.0_152 - /usr/bin/javac
>     Perl: 5.18.4 - /usr/bin/perl
>     PHP: 7.3.11 - /usr/bin/php
>     Python: 2.7.16 - /usr/bin/python
>     Python3: 3.8.3 - /usr/local/opt/python@3.8/bin/python3
>     Ruby: 2.6.3 - /Users/serdarmustafa/.rvm/rubies/ruby-2.6.3/bin/ruby
>   Databases:
>     MySQL: 5.7.29 - /usr/local/opt/mysql@5.7/bin/mysql
>     PostgreSQL: 12.3 - /usr/local/bin/postgres
>     SQLite: 3.32.1 - /usr/local/opt/sqlite/bin/sqlite3
>   Browsers:
>     Chrome: 83.0.4103.106
>     Firefox: 68.0
>     Firefox Developer Edition: 77.0
>     Safari: 13.1.1
>   Monorepos:
>     Lerna: 3.22.1
> ```
> 
> ![Screenshot 2020-06-22 at 14 24 16](https://user-images.githubusercontent.com/45297391/85284837-86557e80-b498-11ea-82c5-9112a639a012.png)

It's actually NOT hung, it's processing. If you add console.logs to migrate.js, you'll see it's processing. You can also check the outputted json folder, to see it's building.
